File: app/src/main/java/com/health/openworkout/core/workout/PikeWalk.java
package com.health.openworkout.core.workout;

import com.health.openworkout.R;
import com.health.openworkout.core.datatypes.WorkoutItem;

public class PikeWalk extends WorkoutItem {
    public PikeWalk() {
        super();
        setName(getContext().getString(R.string.workout_name_pike_walk));
        setDescription(getContext().getString(R.string.workout_description_pike_walk));
        setImagePath("pike_walk.png");
        setVideoPath("pike_walk.mp4");
        setTimeMode(false);
    }
}

File: app/src/main/java/com/health/openworkout/core/workout/PikeWalk.kt
package com.health.openworkout.core.workout

import com.health.openworkout.R
import com.health.openworkout.core.datatypes.WorkoutItem

class PikeWalk : WorkoutItem() {
    init {
        name = context.getString(R.string.workout_name_pike_walk)
        description = context.getString(R.string.workout_description_pike_walk)
        imagePath = "pike_walk.png"
        videoPath = "pike_walk.mp4"
        isTimeMode = false
    }
}
